### PR TITLE
Bump fedora cloud VM to version 34

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
None
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by JAORMX — APPROVED** [review]

**Comment by saschagrunert** [comment]
The selinux init container is complaining:

```
+ semodule -i /usr/share/selinuxd/templates/base_container.cil /usr/share/selinuxd/templates/config_container.cil /usr/share/selinuxd/templates/home_container.cil /usr/share/selinuxd/templates/log_container.cil /usr/share/selinuxd/templates/net_container.cil /usr/share/selinuxd/templates/tmp_container.cil /usr/share/selinuxd/templates/tty_container.cil /usr/share/selinuxd/templates/virt_container.cil /usr/share/selinuxd/templates/x_container.cil
error parsing semanage configuration file: syntax error
semodule:  Could not create semanage handle
+ semodule -i /opt/spo-profiles/selinuxd.cil
error parsing semanage configuration file: syntax error
semodule:  Could not create semanage handle
```

**Comment by saschagrunert** [comment]
Switched to the selinuxd-fedora image for now.

File: test/suite_test.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ const (
 	kindImage        = "kindest/node:v1.20.2"
 	kindDarwinSHA512 = "f2da2d9695ef6d26ff7183f764506223b605bc4638f04742184622ab41599988d0489a54d92ccbad90a5420e47e0868350654d3f8851e0c9afb0fe3539557e00" // nolint: lll
 	kindLinuxSHA512  = "1d2ef9f377ed513850dfd5028c80f3fefd7e3f28e3fc6050593c8a3894b74a40b9761f401ccf44d5aa199e9c753bbb9c1ffb51319f4864a1a2ba184d1574d242" // nolint: lll
+	selinuxdImage    = "quay.io/jaosorior/selinuxd-fedora"
```
Comment (JAORMX) [inline test/suite_test.go:44]: This won't work in OpenShift.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: test/suite_test.go (hunk 1 above)
Comment (saschagrunert) [reply test/suite_test.go]: Reverted the changes to this file.

**Pull request closed without merging**